_posts/2021-09-02-rocket-validator.md

```diff
@@ -0,0 +1,113 @@
+---
+author: Jaime Iniesta
+author_link: https://github.com/jaimeiniesta
+categories: general
+date: 2021-09-02
+layout: post
+title: "Validating Accessibility and HTML with Rocket Validator"
+excerpt: >
+Learn how we're checking and monitoring the new Elixir School site to detect and fix accessibility and HTML issues using Rocket Validator.
+---
+
+## Intro
+
+One of the core principles of Elixir School is to be an inclusive community. We want to help people from all over the world learn to code using Elixir. To facilitate that, we provide translations of the lessons to many languages, maintained by volunteers worldwide. By removing the language barrier, we're helping more people get into the Elixir community.
+
+Apart from the language barrier, there are other barriers that we're aware of - [web accessibility barriers](https://www.w3.org/WAI/people-use-web/abilities-barriers/) are everywhere on the World Wide Web, making it harder for people to access content that is often taken for granted.
+
+Low vision may make a web page unreadable if the text has low contrast or the font size cannot be adapted easily. Unsighted people can use screen readers to hear the page content, but it won't work well if the content is poorly structured and can't be properly interpreted by the screen reader software. People with reduced mobility may prefer to navigate through the lessons using the keyboard instead of the mouse, but again this won't work if the document is not laid out correctly.
+
+The good news is that we can make Elixir School accessible for everyone. And it's not a hard task, it just requires that we're aware of those barriers and that we follow the web standards to provide well-structured web pages.
```

Comment:
```suggestion
The good news is that we can make Elixir School accessible for everyone. It requires that we're aware of those barriers and follows the web standards to provide well-structured web pages.
```

I would suggest avoiding saying that things "are easy" or that they're not challenging, everyone learns at its own pace, and that's fine. See https://justsimply.dev/ for more examples.